modules/tinymce/src/plugins/help/main/ts/ui/PluginsTab.ts

```diff
@@ -7,13 +7,24 @@ import I18n from 'tinymce/core/api/util/I18n';
 import * as Options from '../api/Options';
 import * as PluginUrls from '../data/PluginUrls';
 
+interface PluginData {
+  // The name is just used for sorting alphabetically.
+  name: string;
+  html: string;
```

Comment:
```suggestion
  readonly name: string;
  readonly html: string;
```

Reply:
Sorry. Pointing this out every time should eventually change my muscle memory.

Reply:
Didn't update before rebasing... fixing now.